ofctrl/fgraphFlow.go

```diff
@@ -55,6 +56,8 @@ type FlowAction struct {
 	actionType   string           // Type of action "setVlan", "setMetadata"
 	vlanId       uint16           // Vlan Id in case of "setVlan"
 	macAddr      net.HardwareAddr // Mac address to set
+	ipAddr       net.IP           // IP address to be set
+	l4Port       uint16           // Transport port to be set
```

Comment:
This will prevent us from setting both source and dest ports in the same action. We should preferably create separate fields for source and dest ports..
same comment for src/dest ip address..